docs/products/access/topics/dum-report/index.md

```diff
@@ -1,11 +1,153 @@
-Applications can use this information to provide fine-grained access to data.
+Delegated Administration for Business Partners ([DABP](../../../../dabp/index.md)) keeps information about group memberships, permissions, resources,
+and policies. Applications can use this information to provide fine-grained access to data. Applications can obtain this information via the
+OneWelcome's OAuth and OpenID Connect endpoints.
+
+DABP can respond with a v2 or a legacy v1 version of the report. This depends on your [web client configuration](../web-clients/web-client-configuration.md)
+for the `Group permissions version` field. It will default to v2.
+
+## Person report v2
+
+The [DABP person report](../../../../dabp/guides/technical/person-report.md) for the user is returned via the
+claim `urn:onegini.com:oidc:group_policies` in:
+
+* [ID token](../tokens/id-token.md)
+* [Userinfo](../../api-reference/oidc/user-info.md) response
+* [Token introspection](../../api-reference/token-introspection.md) response
+
+It will never be returned in the access token for v2.
+
+The format of this claim is a JSON object.
+
+Example of this claim:
+
+```json
+{
+  "urn:onegini.com:oidc:group_policies": {
+    "groups": [
+      {
+        "id": "2374b2db-e690-4f3a-89e0-ccd5aaf6c601",
+        "name": "Intermediary A",
+        "policies": [
+          {
+            "id": "77195a4e-c610-4716-b790-721a5fdde1e6",
+            "name": "Pension"
+          },
+          {
+            "id": "6fbe9789-ee84-46eb-9234-2d2d711a0328",
+            "name": "Insurance"
+          }
+        ],
+        "attributes": [
+          {
+            "name": "salesforceId",
+            "value": "12345"
+          }
+        ]
+      },
+      {
+        "id": "2374b2db-e690-4f3a-89e0-ccd5aaf6c601",
+        "name": "Intermediary B",
+        "policies": [
+          {
+            "name": "role_superuser"
+          },
+          {
+            "id": "77195a4e-c610-4716-b790-721a5fdde1e6",
+            "name": "Pension"
+          },
+          {
+            "id": "6fbe9789-ee84-46eb-9234-2d2d711a0328",
+            "name": "Insurance"
+          }
+        ],
+        "attributes": [],
+        "resources": [
+          {
+            "id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+            "name": "string",
+            "externalId": "123456",
+            "privilege": "read",
+            "resourceType": {
+              "id": "3fa85f64-5717-4562-b3fc-2c963f66afa6",
+              "name": "home insurance"
+            }
+          }
+        ]
+      }
+    ]
+  }
+}
+```
+
+## Legacy person report v1
+
+!!! Warning
+This format should not be used for new implementations.
+
+The legacy DABP person report is returned via the claim `group_permissions` in:
+
+* JWT access token. It will not be present if the access token will become too large to be used as bearer
+  token.
```

Comment:
```suggestion
* [JWT access token](../tokens/access-token.md#json-web-token-jwt). It will **not** be present if the access token becomes too large to be used as a bearer
  token.
```